src/test/java/io/confluent/examples/streams/GenericAvroIntegrationTest.java

```diff
@@ -72,7 +70,7 @@ public void shouldRoundTripGenericAvroDataThroughKafka() throws Exception {
     record.put("user", "alice");
     record.put("is_new", true);
     record.put("content", "lorem ipsum");
-    final List<GenericRecord> inputValues = Collections.singletonList(record);
+    final List<Object> inputValues = Collections.singletonList(record);
```

Comment:
new KIP-470 API cannot resolve `GenericRecord` as type, because `KafkaAvroSerializer` uses `Object`...

Reply:
Huh, that's a bummer.